### ISSUE TITLE
request for pivot and stat.describe function.

### ISSUE THREAD

**Issue by cailiang9** [Gmousse/dataframe-js#17]

**Comment by Gmousse** [comment]
Thanks for this submission. @cailiang9 

About .describe(), there is already DataFrame.stat.stats() which gives a similar return than [spark describe method](http://spark.apache.org/docs/1.6.2/api/python/pyspark.sql.html#pyspark.sql.DataFrame.describe).
But the name is not well defined and I think to add an alias (describe is perfect). 

About .pivot() I was thinking about this method since few weeks. 
I will see to work on it quickly. It can be interesting.

**Comment by Gmousse** [comment]
I m working on the pivot method.

Here an example of the api:
````javascript
    df.groupBy('id').pivot('id2', values => values.stat.sum('value')).show();
````

````
BEFORE:


        | id        | id2       | value     |
        ------------------------------------
        | 3         | a         | 1         |
        | 3         | b         | 2         |
        | 1         | a         | 0         |
        | 8         | c         | 1         |


PIVOTED:

        | id        | a         | b         | c         |
        ------------------------------------------------
        | 3         | 1         | 2         | undefined |
        | 1         | 0         | undefined | undefined |
        | 8         | undefined | undefined | 1         |

````

Available this week end.

**Comment by Gmousse** [comment]
The pivot method and the melt method are now available in the 1.2.0.
I close the issue.
